mne_bids/utils.py

```diff
@@ -358,6 +358,10 @@ def get_entity_vals(bids_root, entity_key, *, ignore_sub='emptyroom',
     >>> get_entity_vals(bids_root, entity_key)
     ['05', '06', '07', '08', '09', '10', '11']
 
+    Notes
+    -----
+    This function will scan the entire ``bids_root``, except for a
+    ``derivatives`` subfolder placed directly under ``bids_root``.
```

Comment:
I'm sorry I'm late to the party. Why do you make this "exception" rather than look at `.bidsignore` ?

Reply:
see: https://github.com/bids-standard/bids-validator#bidsignore. Unfortunately it's not mentioned in the specification :-( I think there was some disagreement whether there should be software specific ignore files or a general ignore file.

Reply:
you probably mean https://github.com/bids-standard/bids-specification/issues/131 @jasmainak

Reply:
This sounds interesting! But as long as it's not part of the BIDS standard, … :/

Reply:
whenever we have a use case, it may be worth reviving that discussion :-) 

but big discussions like that (in BIDS) usually need at least one person who REALLY wants stuff to happen ... because otherwise it just goes stale ...

Reply:
oh God yes, let's not go there again :-) 

My personal opinion is that it's safe to exclude anything that's in `.bidsignore` because it has not been validated, and MNE-BIDS assumes that the data is BIDS-valid. Maybe other applications assume that full BIDS-compliance is not necessary and they can have `.whateverignore`. So we can locally take the decision without opening that big discussion again !

Reply:
Yes but my dream is that ideally, the `derivatives/` we create will be just another root of a valid BIDS dataset

Reply:
that's probably already happening ... but I didn't follow very closely :)

Reply:
yes, there is a big validator project to track the slow move to having BIDS-validated derivatives: https://github.com/bids-standard/bids-validator/projects/1

but note: it will always be possible to have "random" derivatives (unvalidated) as part of a BIDS raw directory ... only as soon as you put at `dataset_description.json` into the derivatives directory, it will get validated. (once we are there in the future)